.drone.yml

```diff
@@ -86,7 +86,7 @@ steps:
         - sha512
       draft: true
       files:
-        - _out/*
+        - _out/bootstrap-talos/*/*
```

Comment:
This, and the Dockerfile changes match up to what we do in sidero. Also makes it easy to tweak your clusterctl.yaml file to point to the TAG that you're currently working on.